ids/format.go

```diff
@@ -0,0 +1,40 @@
+package ids
```

Comment:
why not put this in `id.go` ? it's only 40 LOC and the other formatting stuff are there.

Reply:
It would also need to be in `short.go` and then the common implementation would be separated from at least one use case. Here they naturally stay together and provide the reader with context.